apps/frontend/src/pages/chats/chats.tsx

```diff
@@ -47,6 +48,10 @@ const Chats: React.FC = () => {
 
 	const onSubmit = useCallback(
 		(payload: typeof DEFAULT_MESSAGE_PAYLOAD): void => {
+			if (payload.message.trim().length === EMPTY_ARRAY_LENGTH) {
```

Comment:
It is a bit unnecessary to compare to `EMPTY_ARRAY_LENGTH`
```suggestion
			if (!payload.message.trim().length) {
```

Reply:
<img width="783" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/assets/110524091/f1392df7-ebba-43c3-a9dd-a79024f63d8a">

Reply:
You miss "!". Try it.
![hereTry](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/assets/141722848/48e4fd98-aced-4d9a-be80-9e468465e791)

Reply:
<img width="800" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/assets/110524091/b1a8b142-de6d-4815-96d3-01f79f9659e9">